### PR TITLE
Rebrand /multipass landing page

### PR DESCRIPTION
**Pull request by britneywwc**
## Done

- Rebranded multipass landing page
- Migrate from multipass.run to c.com/multipass
- Create secondary navigation for multipass bubble
- Reordered secondary-navigation.yaml by alphabetical order

## QA

- Go to https://canonical-com-1437.demos.haus/multipass
- Check against design file and copy doc
- [Design](https://www.figma.com/design/LGm1MP262JtWfrCxQ3MvqQ/Multipass-Bubble?node-id=1-10105&node-type=instance&m=dev) and [copy docs](https://docs.google.com/document/d/1uI5hsuhe5VUpFSSiQas6FVB2F7RhlbNXcr5Pvr_Lm2c/edit?tab=t.0)
- See that secondary navigation matches https://multipass.run/
- Click on Discourse and see that it links to the discourse forum for multipass
- The other links on the secondary nav are not live yet so will return 404

## Issue / Card

Fixes [WD-16605](https://warthogs.atlassian.net/browse/WD-16605) and [WD-16620](https://warthogs.atlassian.net/browse/WD-16620)

## Screenshots

[if relevant, include a screenshot]

[WD-16605]: https://warthogs.atlassian.net/browse/WD-16605?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[WD-16620]: https://warthogs.atlassian.net/browse/WD-16620?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: secondary-navigation.yaml [hunk 1]
```diff
@@ -81,6 +104,29 @@ mongodb:
...
+  path: /microcloud
+  icon: https://assets.ubuntu.com/v1/6eb42b12-microcloud-logomark.svg
+
+  children:
+    - title: Resources
+      path: /microcloud/resources
+    - title: Docs
+      path: https://canonical-microcloud.readthedocs-hosted.com/en/latest
+
+multipass:
```
Comment (britneywwc) [inline secondary-navigation.yaml:118]: This is the only addition in the file, the other changes are refactoring

**Comment by mattea-turic** [comment]
ty @britneywwc ! a couple comments:

- could the lines here (under "Get started with Ubuntu...", that aren't spanning full-width, be low contrast pls? 
Screenshot 2024-11-27 at 16 56 26

- I just changed it now on the figma file, but in the last section, could you make the "join on github" cta a primary one?
Screenshot 2024-11-27 at 16 57 56

- and a quick question, i see diff padding is used under the buttons e.g. "explore multipass on discourse" and "try ubuntu appliances"; could you use cta block for all of them – or would that cause issues?
Screenshot 2024-11-27 at 17 03 35

**Comment by britneywwc** [comment]
@mattea-turic Thanks for the review, I've applied your suggestions in the recent changes.

**Comment by muhammad-ali-pk** [comment]
LGTM!

**Review by muhammad-ali-pk — APPROVED** [review]
LGTM!

**Pull request merged**